### PR TITLE
Filter by cosign

### PR DESCRIPTION
**Pull request by Andrewjeska**

### CODE REVIEW

**Comment by Andrewjeska** [comment]
```
ALTER TABLE submissions ADD COLUMN "noOfCosigns" INT GENERATED ALWAYS AS (array_length(cosigns, 1)) STORED;
```

TIL you can have derived columns in postgres

File: components/Dropdowns/DropdownRatings.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  React.useEffect(() => {
+    return () => {
+      if (onFocus) onFocus();
+    };
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  return (
+    <div
+      className={classNames("grid grid-cols-1 w-full overflow-y-scroll", {
```
Comment (Timer00) [inline components/Dropdowns/DropdownRatings.tsx:33]: overflow-y-scroll here seems to be unnecessary. In my browsers (chrome, firefox) it displayed an extra white line. 
image

**Comment by Timer00** [comment]
I can't say much about the code, but I tested and it worked! :)

**Comment by Timer00** [comment]
Only feedback I could give is that the UI could be a bit more intuitive, at first I thought you could select multiple ratings and not only one at a time. 
But it's pretty good already as is.

File: components/Dropdowns/DropdownRatings.tsx (hunk 1 above)
Comment (Timer00) [reply components/Dropdowns/DropdownRatings.tsx:33]: Would merge after this small fix

**Comment by Timer00** [comment]
Would merge after fixing issues with the double scrollbars 
image

**Pull request merged**